MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -280,6 +280,43 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 
+	func test_loadFeedWithError_showsErrorOnCompletion() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+
+		XCTAssertNil(sut.errorMessage)
+
+		loader.completeFeedLoadingWithError()
+		XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"))
+	}
+
+	func test_hidesErrorView_onTappingErrorView() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+
+		XCTAssertNil(sut.errorMessage)
+
+		loader.completeFeedLoadingWithError()
+
+		sut.simulateTapOnErrorMessage()
+		XCTAssertNil(sut.errorMessage)
+	}
+
+	func test_reloadingFeed_hidesErrorView() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+
+		XCTAssertNil(sut.errorMessage)
+
+		loader.completeFeedLoadingWithError()
+
+		sut.simulateUserInitiatedFeedReload()
```

Comment:
Same comment as above

```suggestion
		loader.completeFeedLoadingWithError()

		XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"))
		sut.simulateUserInitiatedFeedReload()
```

Reply:
done